src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
```createIndexIfNotExistPostgres``` doesn't ignore the order from the ```indexRef``` list. The ```getIndexColumns``` in that method creates the sql string with the column names along with the order. The default is "ASC" if no order is specified.